src/query/catalog/src/plan/datasource/datasource_info/stage.rs

```diff
@@ -42,9 +46,11 @@ pub struct StageTableInfo {
     // - may need to be purged as well (depends on the copy options)
     pub duplicated_files_detected: Vec<String>,
     pub is_select: bool,
-    pub copy_into_location_options: CopyIntoLocationOptions,
     pub copy_into_table_options: CopyIntoTableOptions,
-    pub stage_root: String,
+
+    // copy into location only
+    pub copy_into_location_ordered: bool,
+    pub copy_into_location_options: CopyIntoLocationOptions,
```

Comment:
```suggestion
    pub copy_into_location_ordered: bool,
    pub copy_into_location_options: CopyIntoLocationOptions,
```
This structure marked as Serialize/Deserialize, will this new field break existing compatibility?

Reply:
no problem since it is only for transfer, not stored